Hedgehog.sln

```diff
@@ -25,18 +27,22 @@ Global
 		{0864FFED-D0B3-4D32-BD38-DB696B7FC5F3}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{0864FFED-D0B3-4D32-BD38-DB696B7FC5F3}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{0864FFED-D0B3-4D32-BD38-DB696B7FC5F3}.Release|Any CPU.Build.0 = Release|Any CPU
-		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Release|Any CPU.Build.0 = Release|Any CPU
-		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Release|Any CPU.Build.0 = Release|Any CPU
 		{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{7E2DE9D9-3831-46D7-B80F-B643E9D53FD3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{48F11B6C-DB48-4244-8575-A4FE9F006E5F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{48F11B6C-DB48-4244-8575-A4FE9F006E5F}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{48F11B6C-DB48-4244-8575-A4FE9F006E5F}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{48F11B6C-DB48-4244-8575-A4FE9F006E5F}.Release|Any CPU.Build.0 = Release|Any CPU
+		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{952547C5-7622-45C8-BB3C-4D1BA2DB747D}.Release|Any CPU.Build.0 = Release|Any CPU
+		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{D38AB54A-D5E8-4B50-927C-472015473AFC}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
This got re-ordered when adding `Hedgehog.DieHarder`.

Reply:
👍